src/PIL/ImageFont.py

```diff
@@ -588,6 +589,7 @@ def getmask(
         stroke_width=0,
         anchor=None,
         ink=0,
+        start=None,
```

Comment:
Needs adding to the docstring below so it'll show up at https://pillow.readthedocs.io/en/latest/reference/ImageFont.html#PIL.ImageFont.FreeTypeFont.getmask

Reply:
Ok, done - https://pillow--6722.org.readthedocs.build/en/6722/reference/ImageFont.html#PIL.ImageFont.FreeTypeFont.getmask